README.md

```diff
@@ -5,48 +5,19 @@ allows you to run services written against the Compute@Edge APIs on your local
 development machine, and allows you to configure testing backends for your
 service to communicate with.
 
-In the near future, Viceroy will be packaged as part of the Fastly CLI and
-integrate with other Compute@Edge workflows. But for the moment, you can use it
-by hand, as described below.
+Viceroy is normally used through the [Fastly CLI's `fastly compute serve`
+command][cli], where it it fully integrated into the Compute@Edge workflows.
+However, it is also a standalone open source tool with its own CLI and a
+Rust library that can be embedded into your own testing infrastructure.
+
+[cli]: https://developer.fastly.com/learning/compute/testing/#running-a-local-testing-server
 
 ## Installation
 
 To install Viceroy, you'll need to first
 [install Rust](https://www.rust-lang.org/tools/install) if you haven't already.
```

Comment:
:grey_question: What would you think about breaking this into two sections: (a) "_building from source_", which would be what's already here, and (b) "_releases_"  mostly just pointing to our [release page](https://github.com/fastly/Viceroy/releases)?

It might also be worth mentioning in this section that the Fastly CLI will install this for you automatically. These instructions are just for people hoping to use it independently of the CLI, e.g. within testing infrastructure.

Reply:
Hmm I have mixed feelings about keeping around the old docs re: installation.

I think part of what's giving me pause is: I don't really know who our audience is, here. We generally don't want customers to use Viceroy directly. OTOH folks hacking directly on it probably already know relevant Rust workflows around installation etc.

Do you have thoughts on audience/goals here?

Reply:
> **[..]** keeping around the old docs **[..]**

Oh my goodness pardon me! To clarify, when I said "_what's already here_" I meant the state of our readme _after_ these changes.

As for audience, I agree that we don't expect (_or want?_) people to use Viceroy directly, without good reason. So I'd hope that this section redirects the general majority of people that will (most likely) need to upgrade their CLI rather than install this tool directly.

The people sticking around after that disclaimer we might assume have some specific use-case that warrants using Viceroy directly, and can then choose from either a release binary or building/installing from source via `cargo install`.

How's that sound?

Reply:
Building on this: maybe we want to name-drop that people will want a CLI version >= `0.34.0`, and then point them [here](https://github.com/fastly/cli/releases) if they need to upgrade.

Reply:
Ahhh sorry for the misunderstanding! Yes, this makes a ton of sense, I will update accordingly.

Reply:
No apology needed, thank you for asking the question and forcing me to hash this out a bit more explicitly! We're better off for it. :blush: